src/v/rpc/logger.cc

```diff
@@ -10,5 +10,5 @@
 #include "rpc/logger.h"
```

Comment:
nit: the .cc file can be removed (probably this is already planned but anyway)

Reply:
will do. I still haven't figured out how to fix this dependency issue yet

Reply:
fixed